plugins/api-authentication/token/services/provider.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * @package     Joomla.Plugin
+ * @subpackage  Apiauthentication.token
+ *
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Extension\PluginInterface;
+use Joomla\CMS\Plugin\PluginHelper;
+use Joomla\CMS\User\UserFactoryInterface;
+use Joomla\Database\DatabaseInterface;
+use Joomla\DI\Container;
+use Joomla\DI\ServiceProviderInterface;
+use Joomla\Event\DispatcherInterface;
+use Joomla\Filter\InputFilter;
+use Joomla\Plugin\APIAuthentication\Token\Extension\Token;
+
+return new class implements ServiceProviderInterface {
+	/**
+	 * Registers the service provider with a DI container.
+	 *
+	 * @param   Container  $container  The DI container.
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function register(Container $container)
+	{
+		$container->set(
+			PluginInterface::class,
+			function (Container $container)
+			{
+				$plugin = new Token(
+					$container->get(DispatcherInterface::class),
+					(array) PluginHelper::getPlugin('api-authentication', 'token'),
+					$container->get(UserFactoryInterface::class),
+					new InputFilter()
```

Comment:
```suggestion
					new InputFilter
```
https://ci.joomla.org/joomla/joomla-cms/54234/1/6